src/components/header/renderer.tsx

```diff
@@ -875,17 +845,19 @@ class Header extends React.Component<Props, State> {
           </PortalWithState>
 
           {this.state.helpModalOpen && (
-            <div className="modal-background" onClick={this.handleHelpModalCloseClick.bind(this)}>
-              <div className="modal modal-top" onClick={e => e.stopPropagation()}>
-                <div className="modal-header">
-                  <button className="close-button" onClick={this.handleHelpModalCloseClick.bind(this)}>
-                    <X />
-                  </button>
+            <Portal>
```

Comment:
@domoritz Since I can't use `<PortalWithState />` because of custom events, I used `<Portal />`. `<Portal />` doesn't have `closeOnEsc` prop. Therefore, `handleHelpModalCloseEsc` is required.

Reply:
Why do you need custom events?

Reply:
To add <kbd>Ctrl</kbd> + <kbd>?</kbd> as a shortcut. React only has `onClick`, `onKeyPress` and so on.